Mathlib/Tactic/Linter/DeprecatedSyntaxLinter.lean

```diff
@@ -62,11 +62,40 @@ register_option linter.style.admit : Bool := {
   descr := "enable the admit linter"
 }
 
+/-- The option `linter.style.maxHeartbeats` of the deprecated syntax linter flags usages of
+`set_option maxHeartbeats n in cmd` that do not add a comment explaining the reason for the
+modification of the maxHeartbeats. -/
+register_option linter.style.maxHeartbeats : Bool := {
+  defValue := false
+  descr := "enable the maxHeartbeats linter"
+}
+
+/-- If the input syntax is of the form `set_option <option> num in <string> cmd`,
+where `<option>` contains `maxHeartbeats`, then it returns
+* the `<option>` as a name (typically, `maxHeartbeats` or `synthInstance.maxHeartbeats`);
```

Comment:
Just a grammar nerd question, not important: my (non-native speaker) intuition would be to use a comma. Do you know a rule which I don't, then I'll happily learn :-)

Reply:
I am not sure: I added a comma.